cloud/scope/vpc_cluster.go

```diff
@@ -486,9 +497,186 @@ func (s *VPCClusterScope) createVPC() (*vpcv1.VPC, error) {
 	} else if vpcDetails == nil {
 		return nil, fmt.Errorf("no vpc details after creation")
 	}
-	if err = s.TagResource(s.IBMVPCCluster.Name, *vpcDetails.CRN); err != nil {
+
+	// NOTE: This tagging is only attempted once. We may wish to refactor in case this single attempt fails.
+	if err = s.TagResource(s.Name(), *vpcDetails.CRN); err != nil {
```

Comment:
Do we need to make this change as part of this PR?
It seems not relevant to me. Please see whether we can create a separate issue and handle it separately.

Reply:
I can create a second PR, if that PR can be reviewed and merged prior to this PR, to fix this invalid reference.

Reply:
I have the separate PR opened for the necessary fixes.
https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/1949